clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java

```diff
@@ -252,6 +252,10 @@ public TitusHealth getHealth() {
   }
 
   private List<Job> getJobs(JobQuery.Builder jobQuery) {
+    return getJobs(jobQuery, true);
+  }
+
+  private List<Job> getJobs(JobQuery.Builder jobQuery, boolean includeTasks) {
```

Comment:
Neato -- want to also add an `includeTasks` for `findJobByName`?

The current call in `TitusClusterCachingAgent` could be swapped over to make use of it.

Also happy to do this as a subsequent commit.